doc/connected-components.adoc

```diff
@@ -12,10 +18,79 @@ _Connected Components_ or _UnionFind_ basically finds sets of connected nodes wh
 
 [source,cypher]
 ----
+CALL algo.unionFind.stream('', '', {property:'cost', defaultValue:10.0, threshold:5.0}) 
+YIELD nodeId,setId
+----
+
+[source,cypher]
+----
+CALL algo.unionFind('', '', {property:'cost', defaultValue:10.0, threshold:5.0, write:true, clusterProperty:"cluster"}) 
+YIELD nodeCount, setCount, loadDuration, evalDuration, writeDuration
 ----
 
 == Example Usage
 
+== Syntax
+
+.Running algorithm and writing back results
+[source,cypher]
+----
+CALL algo.unionFind(label:String, relationship:String, {property:'propertyName', threshold:0.42,
```

Comment:
should change also the `property` to `weightProperty`, just spoke with martin and he said that whenever the default value for `property` is cost or the name of the param is `costProperty` it means the same as `weightProperty`. So I would stick with `weightProperty`